coq-builtin.elpi

```diff
@@ -969,7 +969,7 @@ type never simplification_strategy. % Arguments foo : simpl never
 type when list int -> option int ->
           simplification_strategy. % Arguments foo .. / .. ! ..
 type when-nomatch list int -> option int ->
-                  simplification_strategy. % Arguments foo .. / .. ! .. : simpl moatch
+                  simplification_strategy. % Arguments foo .. / .. ! .. : simpl nomatch
```

Comment:
This file is generated from src/coq_elpi_builtins.ml

Reply:
Oups, I push a patch to the right file